src/main/scala/io/opentargets/etl/backend/Drug.scala

```diff
@@ -190,6 +187,7 @@ object DrugHelpers {
           |  array_distinct(transform(mechanisms_of_action, x -> x.target_type)) as uniqueTargetTypes), null) as mechanismsOfAction
           |""".stripMargin
 
+      // drug rewrite: done
```

Comment:
@JarrodBaker are these comments essentials? If these are not needed please remove from the code

Reply:
Sorry, these were comments to myself which I accidentally committed.